### PR TITLE
fixed tests

### PR DESCRIPTION
**Pull request by mandd**
--------
Pull Request Description
--------
##### What are the significant changes in functionality due to this change request?

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: framework/PostProcessors/InterfacedPostProcessor.py [hunk 1]
```diff
@@ -195,5 +195,7 @@ def collectOutput(self, finishedJob, output):
     if isinstance(evaluations, Runners.Error):
       self.raiseAnError(RuntimeError, "No available output to collect (run possibly not finished yet)")
     evaluation = evaluations[1]
-    output.addExpectedMeta(set(self.metaKeys))
+    print(evaluation)
```
Comment (wangcj05) [inline framework/PostProcessors/InterfacedPostProcessor.py]: remove print

File: framework/PostProcessors/InterfacedPostProcessor.py [hunk 2]
```diff
@@ -195,5 +195,7 @@ def collectOutput(self, finishedJob, output):
...
+    for key in evaluation['data'].keys():
+      print(key, type(evaluation['data'][key]))
```
Comment (wangcj05) [inline framework/PostProcessors/InterfacedPostProcessor.py]: remove print

File: tests/framework/PostProcessors/InterfacedPostProcessor/gold/HS2PS/PrintPPPointSet_dump.csv [hunk 3]
```diff
@@ -1,3 +1,3 @@
-y0,x0,z0,1,0,3,2,5,4
+y0,x0,z0,3,2,1,0,5,4
```
Comment (wangcj05) [inline tests/framework/PostProcessors/InterfacedPostProcessor/gold/HS2PS/PrintPPPointSet_dump.csv]: why only the label changes, not the actual value?

**Comment by PaulTalbot-INL** [comment]
It looks like there's a trailing whitespace issue in TypicalHistory.

**Comment by wangcj05** [comment]
Just need the test results on Falcon.

**Comment by wangcj05** [comment]
Test results on Falcon:

```
*****************
Number of success first file:
352
Number of success second file:
354
NUMBER OF SUCCESSES INCREASED: +2
**** NUMBER OF TESTS IN THE TWO FILES ARE DIFFERENT: ****
**** first file :536 /= second file: 532****
TOTAL TIME OF TESTS INCREASED       : reference_time(499.5) < new_time(532.2)
AVERAGE TIMES FOR SUCCESS TESTS ARE : reference_time(1.41903409091) v.s. new_time(1.50338983051)
*********************************
```

**Pull request merged**